packages/frontend/src/pages/admin/ads.vue

```diff
@@ -47,8 +47,9 @@
 			<MkButton class="button" @click="more()">
 				<i class="ti ti-reload"></i>{{ i18n.ts.more }}
 			</MkButton>
-		</MkSpacer>
-	</MkStickyContainer>
+		</div>
+	</MkSpacer>
+</MkStickyContainer>
```

Comment:
Looks like there is a compile error in `ads.vue` starting from 543ba2b. This is just a workaround.